lib/stages/hdf5FileWrite.cpp

```diff
@@ -0,0 +1,240 @@
+#include "hdf5FileWrite.hpp"
+
+#include "Config.hpp"          // for Config
+#include "StageFactory.hpp"    // for REGISTER_KOTEKAN_STAGE, StageMakerTemplate
+#include "buffer.hpp"          // for Buffer, get_metadata_container, mark_frame_empty, regis...
+#include "bufferContainer.hpp" // for bufferContainer
+#include "chimeMetadata.hpp"   // for chimeMetadata
+#include "chordMetadata.hpp"   // for chordMetadata
+#include "errors.h"
+#include "kotekanLogging.hpp"    // for ERROR, INFO
+#include "metadata.h"            // for metadataContainer
+#include "prometheusMetrics.hpp" // for Metrics, Gauge
+#include "visUtil.hpp"           // for current_time
+
+#undef NDEBUG
```

Comment:
Just want to make sure this is here intentionally!

Reply:
I replaces all the asserts by proper error messages.